core/src/main/java/edu/wpi/grip/core/StartStoppable.java

```diff
@@ -0,0 +1,52 @@
+package edu.wpi.grip.core;
+
+
+import com.google.common.eventbus.EventBus;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * An Object that can be stopped and started multiple times.
+ */
+public interface StartStoppable {
+
+    /**
+     * Starts this source.
+     *
+     * @return The source object that created the camera
+     * @throws IOException If the source fails to be started
+     */
+    default <T extends StartStoppable> T start(EventBus eventBus) throws IOException {
+        start();
+        eventBus.register(this);
+        return (T) this;
+    }
```

Comment:
This can be removed once dependency injection is added. Put it on your 'hit list' for methods to remove. Just merge it for now because it makes the code cleaner.